resolwe_bio/processes/plots/dexseq_plot.yml

```diff
@@ -0,0 +1,75 @@
+# =============
+# Plot - DEXSeq
+# =============
+---
+
+- slug: plot-dexeq
+  name: DEXSeq - Plot
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:2.1.1
+  data_name: 'Plot - DEXSeq (case vs. control)'
```

Comment:
Include the `gene` input variable in the object name to make it more informative in BD.